docs/getting_started.md

```diff
@@ -2,12 +2,37 @@
 
 ## Installation
 
-Install Cubed with pip:
+### Conda
+
+You can install cubed with a minimal set of dependencies using conda:
+
+```shell
+conda install -c conda-forge cubed
+```
+
+### Pip
+
+You can also install cubed with pip:
 
 ```shell
 python -m pip install cubed
 ```
 
+Cubed has many optional dependencies, which can be installed in sets for different functionality (especially for running on different executors):
+
+    $ python -m pip install "cubed[diagnostics]"  # Install optional dependencies for cubed diagnostics
+    $ python -m pip install "cubed[beam]"         # Install optional dependencies for the beam executor
+    $ python -m pip install "cubed[lithops]"      # Install optional dependencies for the lithops executor
+    $ python -m pip install "cubed[modal]"        # Install optional dependencies for the modal executor
+    $ python -m pip install "cubed[complete]"     # Install all of the above
+
+To see the full list of which packages are installed with which options see `[project.optional_dependencies]` in `pyproject.toml`:
+
+.. literalinclude:: ../pyproject.toml
```

Comment:
This didn't render properly when I tried it locally. Does it need something extra installed for Sphinx?

Reply:
Oh oops - I didn't clock that these docs files are .md not .rst :man_facepalming: 

How are you building these exactly? That's something else that should be added to the install instructions / CI.

Reply:
CI runs https://github.com/tomwhite/cubed/blob/main/.github/workflows/docs.yml

To run locally you can create a python env with `./docs/requirements.txt` and then run `make clean html` in the `docs` directory.

Reply:
Thanks - turns out you can just wrap the rst command with `{eval-rst}`.